script/2_PowerSystems_examples/add_forecasts.jl

```diff
@@ -22,14 +26,9 @@ include(joinpath(pkgpath, "test", "2_PowerSystems_examples", "parse_matpower.jl"
 FORECASTS_DIR = joinpath(base_dir, "forecasts", "5bus_ts")
 fname = joinpath(FORECASTS_DIR, "timeseries_pointers_da.json")
 open(fname, "r") do f
-    for line in eachline(f)
-        println(line)
-    end
+    @JSON3.@pretty JSON3.read(f)
 end
 
-# ### Read the pointers
-ts_pointers = PowerSystems.IS.read_time_series_file_metadata(fname)
-
-# ### Read and assign time series to `System` using the `ts_pointers` struct
-add_time_series!(sys, ts_pointers)
+# ### Read and assign time series to `System` using these parameters.
```

Comment:
@claytonpbarrows IMHO it is better to keep this simpler. If you do want to expose the intermediate step to users, I'll revert the change.

Reply:
I think the same, there are too many intermediate steps here for this to be a tutorial